tools/hooks/useDebounce.ts

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from 'react'; // React
+
+// From usehooks.com/useDebounce
+export default function useDebounce<T>(value: T, delay: number) {
```

Comment:
Sorry @EmilioMena looks like we forgot to add the return type for the hook. Could you please add it in a similar fashion to how it's done in our other hooks?